tests/README.md

```diff
@@ -8,91 +8,223 @@ Priority is to provide very simple system of test scenarios, easily extendable
+  - `openssh-clients`
+  - `libvirt`
+  - `libvirt-daemon`
+  - `python2-libvirt/python3-libvirt`
+  - `virt-install`      (recommended, used by `install_vm.py` script)
+  - `libvirt-client`    (optional, to manage VMs via console)
+  - `virt-manager`      (optional, to manage VMs via GUI)
+  - `virt-viewer`       (optional, to access graphical console of VMs)
+  - `ansible`           (required if remediating via Ansible)
+- A VM that fullfils following requirements:
+  - Package `qemu-guest-agent` installed
+  - Package `openscap` version 1.2.15 or higher installed
+  - `root` can login via ssh (it is recommended to setup key-based authentication)
+  - `root` can install packages (for RHEL7, it means subscription enabled).
+
+An easy way to install your VM is via `install_vm.py` script. It will setup a VM\
+according to the requirements, including configuration of a key for SSH login.
+
+Common usage looks like:
+```
+$ ./install_vm.py --domain test-suite-fedora --distro fedora
+```
+
+By default, the key at `~/.ssh/id_rsa.pub` is used. You can change default key used via `--ssh-pubkey` option.
+
+By default, the VM will be created on the user hypervisor, i.e. `qemu:///session`.
+For TestSuite this should be enough, in case you need the VM to reside under `qemu:///system`, use the install script with `--libvirt qemu:///system`.
+
+When installing a RHEL VM, you will still need to subscribe it.
+
+*TIP*: Create a snapshot as soon as your VM is setup. This way, you can manually revert
+in case the test suite breaks something and fails to revert.\
+Do not use snapshot names starting with `ssg_`.
+
+### Container backends
+
+There are 2 container backends supported, Podman and Docker.
+
+The container image needs to be prepared so it can accept ssh connections from `root`, and run OpenSCAP scan.\
+The image needs to fulfil the following requirements:
+
+  - Package `openssh-server` installed
+    - `root`'s `.ssh/authorized_keys` is setup with correct permissions
+    - the container's server host keys have to be in place
+  - Packages `scp` and `openssh-clients` are installed
+    - `scp` requires more than a ssh server on the server-side
+  - Package `openscap-scanner` version 1.2.15 or higher installed
+  - You may want to include other packages, as base images tend to be bare-bone and tests may require more packages to be present.
+
+You can use `test_suite-*` Dockerfiles in the `content/Dockerfiles` directory to build the images.
+We recommend to use RHEL-based containers, as the test suite is optimized for testing the RHEL content.
+
+#### Podman
+
+To use Podman backend, you need to have:
+
+- `podman` package installed
+
+<!--- try slirp4netns --->
+*NOTE*: With Podman, you have to run all the operations as root. Podman supports rootless containers, but the test suite internally uses a container exposing a TCP port. As of Podman version 0.12.1.2, port bindings are not yet supported by rootless containers.
```

Comment:
The same applies also for docker as you need access to `/var/run/docker.sock` which is owned by a root, therefore you need to use sudo by default.

Reply:
This is mentioned [here](https://github.com/ComplianceAsCode/content/pull/3732/files#diff-62f0ff60ede3b78b9c95624e2f61d6c1R125).

I did not add a to Docker because the `root` situation with Podman should be temporary.